docs/templates.md

```diff
@@ -148,6 +148,22 @@ keys = [
 {{end}}
 ```
 
+#### Traversing the Map created from json
```

Comment:
Let's simplify this to "Advanced Map Traversals" for consistency with other titles.